### PR TITLE
Added BetterPassword version 1.2

### PR DESCRIPTION
**Pull request by wopsononock**

### CODE REVIEW

**Comment by ctgraham** [comment]
Thanks, @wopsononock !

**Comment by asmecher** [comment]
@ctgraham, have you reviewed this?

**Comment by ctgraham** [comment]
> @ctgraham, have you reviewed this?

@asmecher , meet @wopsononock . @wopsononock will be updating each of our plugins to support 3.3. Our process has an internal review within ulsdevteam prior to packaging and releasing the plugin version, then the PR to plugin-gallery.

**Comment by asmecher** [comment]
Ah, great! I figured it was something like that. Good to meet you, @wopsononock! Just to confirm, @ctgraham, has the internal review been completed or does that kick off with this PR?

**Comment by ctgraham** [comment]
Our internal review for this is completed; each review will precede the packaging/release, which will precede the PR here.

**Comment by asmecher** [comment]
Gotcha, so this is ready for merging?

**Comment by ctgraham** [comment]
Found a [bug with the version.xml](https://github.com/ulsdevteam/pkp-betterPassword/issues/29) here, with an upcoming patch.

**Comment by ctgraham** [comment]
@wopsononock , please update this PR with the new md5 hash for the release.

**Comment by ctgraham** [comment]
@wopsononock , this md5 is still not matching.

**Comment by wopsononock** [comment]
I'm calculating the md5 using md5sum betterPassword-v1.2.0-0.tar.gz and coming up with the same thing.  I must be missing something here.

**Comment by ctgraham** [comment]
The download link is a tar.gz, but the [value in the xml is a .tgz](https://github.com/pkp/plugin-gallery/pull/128/files#diff-40e37cab0fab1110313d27d74efc0d9fef18d8bba94682eb6841a45d983147acR4526).  The MD5 which Travis CI is getting must be from a 404.

**Comment by ctgraham** [comment]
@wopsononock , please add 3.3.0-9 and 3.3.0-10 to this PR.

File: plugins.xml [hunk 1]
```diff
@@ -4522,6 +4522,44 @@
...
+			<compatibility application="ojs2">
+				<version>3.3.0.0</version>
+				<version>3.3.0.1</version>
+				<version>3.3.0.2</version>
+				<version>3.3.0.3</version>
+				<version>3.3.0.4</version>
+				<version>3.3.0.5</version>
+				<version>3.3.0.6</version>
+				<version>3.3.0.7</version>
+				<version>3.3.0.8</version>
```
Comment (ctgraham) [inline plugins.xml:4536]: Still missing 3.3.0-9 and 3.3.0-10.

**Review by ctgraham — CHANGES_REQUESTED** [review]

**Pull request merged**